lib/collection/src/operations/point_ops.rs

```diff
@@ -133,10 +142,10 @@ impl Validate for PointsSelector {
 }
 
 // Structure used for deriving custom JsonSchema only
-#[derive(Debug, Deserialize, Serialize, JsonSchema, Validate)]
-struct PointsList {
-    points: Vec<PointStruct>,
-}
+// #[derive(Debug, Deserialize, Serialize, JsonSchema, Validate)]
+// struct PointsList {
+//     points: Vec<PointStruct>,
+// }
```

Comment:
We should probably remove this, we redefine `PointsList` below.

Reply:
check